coreos-base/coreos-init/coreos-init-9999.ebuild

```diff
@@ -2,6 +2,8 @@
 # Copyright (c) 2011 The Chromium OS Authors. All rights reserved.
 # Distributed under the terms of the GNU General Public License v2
 
+# NOTE: if /etc/sshd_config changes make sure to copy the change to sys-auth/google-oslogin
```

Comment:
`/etc/ssh/sshd_config`?

I'm concerned that these comments are too subtle and will be missed.  I understand not putting them in the files themselves, but how about adding `README`s to the package directories?

Reply:
In addition to or instead of?

Reply:
Instead of, I think?  Either way.

Reply:
:+1: I'm in favor of that as well. Means we don't have to keep the notes about keeping things in sync in sync

Reply:
Fixed.